05_stringer/undrewb/main_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	want := strconv.Quote("127.0.0.1\n")
+	got := strconv.Quote(buf.String())
+
+	assert.Equal(t, want, got)
+}
+
+var stringerData = []struct {
+	name  string
+	input IPAddr
+	want  string
+}{
+	{
+		name:  "localhost",
+		input: IPAddr{127, 0, 0, 1},
+		want:  "127.0.0.1",
+	},
+	{
+		name:  "four octet addr",
+		input: IPAddr{68, 2, 44, 125},
+		want:  "68.2.44.125",
+	},
+	{name: "three octet addr",
```

Comment:
The indentation on this is inconsistent with the rest, can you please fix this so this line starts on the one below